lunatrace/bsl/backend/src/config.ts

```diff
@@ -77,12 +94,45 @@ export function getEtlBucketConfig(): SbomHandlerConfig {
   };
 }
 
-export function getQueueHandlerConfig(): QueueHandlerConfig {
-  const queueName = checkEnvVar('QUEUE_NAME');
+export function getQueueHandlerConfig(): QueueHandlerWorkerConfig {
   const handlerName = checkEnvVar('QUEUE_HANDLER');
+
+  const DEFAULT_QUEUE_MAX_MESSAGES = 10;
+  const DEFAULT_QUEUE_VISIBILITY = 60;
+
+  const handlerConfigLookup: Record<QueueHandlerType, QueueHandlerConfig> = {
+    'process-webhook': {
+      maxMessages: 1,
+      visibility: DEFAULT_QUEUE_VISIBILITY * 5,
```

Comment:
We should probably make this shorter now unless you think this will take a while still.